┌ Info: 2024-02-21 11:31:12 PgDBController.create_jetelina_id_sequence() error: LibPQ.Errors.DuplicateTable("ERROR:  relation \"jetelina_table_id_sequence\" already exists\n")
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
┌ Info: 2024-02-21 11:31:15 SQLAnalyzer.main() start with : 48 hr interval
└ @ Jetelina.JLog /home/onoke/Jetelina/Jetelina/app/resources/JLog.jl:71
